### PR TITLE
docs: Explain SIGUSR1 usage in dev guide

### LINKED ISSUE #70: Document SIGUSR1 in developer guide
Explain how to obtain a backtrace by sending `SIGUSR1` to a kata component.

### PR DESCRIPTION
**Pull request by jodh-intel**
Add details to the developer guide of how to obtain a backtrace by
sending a `SIGUSR1` signal to the component.

Fixes #70.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Hi @egernst, @klynnrif - ptal.

**Comment by jodh-intel** [comment]
Related: https://github.com/kata-containers/agent/pull/224.

File: Developer-Guide.md [hunk 1]
```diff
@@ -275,7 +275,19 @@ script and paste its output directly into a
 
 To perform analysis on Kata logs, use the
 [`kata-log-parser`](https://github.com/kata-containers/tests/tree/master/cmd/log-parser)
-tool.
+tool, which can convert the logs into formats (such as JSON, TOML, XML, and YAML).
```
Comment (klynnrif) [inline Developer-Guide.md]: tool, which can convert the logs into formats (e.g. JSON, TOML, XML, and YAML).

Comment (jodh-intel) [reply Developer-Guide.md]: Thanks @klynnrif - branch updated.

**Comment by egernst** [comment]
@klynnrif - heads up -- should be ready for your approval now.

**Pull request merged**